src/ReactPixiFiber.js

```diff
@@ -142,12 +165,45 @@ const ReactPixiFiber = ReactFiberReconciler({
         );
         break;
       default:
+        if (type in INJECTED_TYPES) {
+          const injectedType = INJECTED_TYPES[type];
+          let customDisplayObject;
+          if (typeof injectedType === "function") {
+            customDisplayObject = injectedType;
+          } else if (typeof injectedType.customDisplayObject === "function") {
+            customDisplayObject = injectedType.customDisplayObject;
+          }
+
+          invariant(
+            customDisplayObject,
+            'Invalid Component injected to ReactPixiFiber: "%s".',
+            type
+          );
+
+          instance = customDisplayObject(props);
+
+          if (typeof injectedType.customApplyProps === "function") {
+            instance._customApplyProps = injectedType.customApplyProps.bind({
+              applyDisplayObjectProps: defaultApplyProps.bind(null, instance)
+            });
+          }
+          if (typeof injectedType.customDidAttach === "function") {
+            instance._customDidAttach = injectedType.customDidAttach;
+          }
+          if (typeof injectedType.customWillDetach === "function") {
+            instance._customWillDetach = injectedType.customWillDetach;
+          }
+        }
         break;
     }
 
-    invariant(instance, 'ReactPixiFiber does not support the type: "%s"', type);
+    invariant(
+      instance,
+      'ReactPixiFiber does not support the type: "%s".',
+      type
+    );
 
-    applyProps(instance, props);
+    applyProps(instance, {}, props);
```

Comment:
I think it's better to pass empty object here than infer an `undefined`, in case of destructuring an object (which would fail with undefined value).